contributing.md

```diff
@@ -3,52 +3,90 @@
 Our Commitment to Open Source can be found [here](https://zeit.co/blog/oss)
 
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
-1. Install yarn: `npm install -g yarn`
-1. Install the dependencies: `yarn`
-1. Run `yarn run dev` to build and watch for code changes
+2. Install yarn: `npm install -g yarn`
+3. Install the dependencies: `yarn`
+4. Run `yarn run dev` to build and watch for code changes
+5. The development branch is `canary`. On a release, the relevant parts of the changes in the `canary` branch are rebased into `master`.
 
 ## To run tests
 
 Running all tests:
 
-```
+```sh
 yarn testonly
 ```
 
 Running a specific test suite inside of the `test/integration` directory:
 
-```
+```sh
 yarn testonly --testPathPattern "production"
 ```
 
 Running just one test in the `production` test suite:
 
-```
+```sh
 yarn testonly --testPathPattern "production" -t "should allow etag header support"
 ```
 
-## Running the integration test apps without running tests
+## Running the integration apps
 
-```
-./node_modules/.bin/next ./test/integration/basic
-```
+The correct path to the compiled `next` binary can be discovered by running:
 
-## Testing in your own app
+```sh
+find . -name next -perm -u=x -type f
```

Comment:
Did the `./node_modules/.bin/next` not exist? I can imagine lerna didn't work correctly and the file was not output 🤔

Reply:
`./packages/next/dist/bin/next` is more correct though, so great change there 💯

Reply:
Yes - the file did not exist 🤔